Emby.Server.Implementations/HttpServer/HttpListenerHost.cs

```diff
@@ -215,27 +205,17 @@ private Exception GetActualException(Exception ex)
 
         private int GetStatusCode(Exception ex)
         {
-            if (ex is ArgumentException)
+            switch (ex)
             {
-                return 400;
+                case ArgumentException _:                 return 400;
+                case SecurityException _:                 return 401;
+                case PaymentRequiredException _:          return 402;
```

Comment:
wtf? :) IMHO this should be `500` as we're not having any payments, so having an exception about one is a server error.

Reply:
There was/is a PR open that removes it.

Reply:
Can you post a PR # here?

Reply:
#412